sccameraview/src/main/java/com/example/sccameraview/Camera1View.java

```diff
@@ -64,7 +63,7 @@ void openCamera() {
             camera.setPreviewTexture(getSurfaceTexture());
             camera.startPreview();
         } catch (IOException e) {
-            Timber.e(e);
+            Log.d("SCCameraView", e.getMessage());
```

Comment:
Please move this to a constant somewhere.

Use the appropriate log level, as with Timber. For example here you should use `Log.e`.